### PR TITLE
Change link for drug stock progress-tab - Part 2

### PR DESCRIPTION
**Pull request by harimohanraj89**
**Story card:** [ch2929](https://app.clubhouse.io/simpledotorg/story/2929/handle-navigation-and-from-drug-stock-success-page-in-the-android-app)

Follow-up to #2153 , adds the deeplink to the back button on the submission form as well.

### CODE REVIEW

File: app/views/webview/drug_stocks/new.html.erb [hunk 1]
```diff
@@ -16,14 +16,14 @@
...
+
         <style>
           label.sr-only {display:none;}
           .form-group {padding-left:0;padding-top:4px;padding-bottom:32px;}
         </style>
-      
+
         <div class="progress-contents">
-          <a href="#" onclick="closeWindow('drug-stock-form', 'progress-start'); return false" class="help-back">
+          <a href="simple://progress-tab" title="Go Back" class="help-back">
```
Comment (prabhanshuguptagit) [inline app/views/webview/drug_stocks/new.html.erb:26]: Do we need to do both or just the new one will do?

**Review by prabhanshuguptagit — APPROVED** [review]

File: app/views/webview/drug_stocks/new.html.erb (hunk 1 above)
Comment (harimohanraj89) [reply app/views/webview/drug_stocks/new.html.erb:26]: Do you mean both Back buttons? We want to attach the functionality to both. Whether you click "Back" on the drug stuck submission form page, or on the success page, you should be directed back to the top-level progress tab.

Comment (prabhanshuguptagit) [reply app/views/webview/drug_stocks/new.html.erb:26]: I was wondering if the same button click triggers both the "clicks" (dumb thought). This makes sense 👍🏼

**Review by rsanheim — APPROVED** [review]
Ah yup, this makes sense 👍

**Pull request merged**